### PR TITLE
delete post tags endpoint testing 

### PR DESCRIPTION
**Pull request by beblicarl**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [x] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [x] All tests are passing.
-   [x] Tested changes locally.
-   [x] Improved documentation or readme where necessary.
-   [x] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

#### What does this PR do?

1. Moved the delete post tag function to a separate file
2. Added test to the delete post tag service
3. Added endpoint testing to the delete article tag endpoint

https://github.com/tbf-challenge/teamwork-be/issues/334
#### How should this be manually tested?

1. Start the test with `npm test`
2. This should return all tests passing 

#### What are the relevant issues?
https://github.com/tbf-challenge/teamwork-be/issues/334

### CODE REVIEW

**Comment by beblicarl** [comment]
@jocrah , @jakazzy Kindly review this PR for me, thanks

**Comment by jakazzy** [comment]
@beblicarl  Please instead of the issue number, lets add a brief description on what changes to expect in the PR

**Comment by beblicarl** [comment]
> @beblicarl Please instead of the issue number, lets add a brief description on what changes to expect in the PR

yes please

File: src/routes/articles/deleteArticleTag.e2e.test.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+				.expect(401)
+
+		})
+        
+
+	})
+
+	describe('Success', () => {
+
+		it('should return 200', async () => {
```
[the remark this answers is not in the document]
Comment (beblicarl) [reply src/routes/articles/deleteArticleTag.e2e.test.js:73]: sure thanks

**Comment by beblicarl** [comment]
> @beblicarl Please instead of the issue number, lets add a brief description on what changes to expect in the PR

done

**Review by jakazzy — APPROVED** [review]

**Pull request merged**